index.bs

```diff
@@ -800,6 +813,14 @@ authorizing an authenticator with which to complete the operation.
         omitted, the RP ID will be set to the [=origin=] specified by the {{WebAuthentication}} object's [=relevant settings
         object=].
 
+    - The <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which are used as an additional
+        constraint on which authenticators are eligible to participate in a [[#makeCredential]] operation.
+        See [[#attachment]] for a description of the attachment values and their meanings.
+
+    - The <dfn>keyStorage</dfn> parameter contains description of where the <a>credential private key</a> should be stored. 
```

Comment:
Could we say something like "The <dfn>keyStorage</dfn> parameter indicates the [RP]'s requirements regarding availability of the credential."?